app/domain/authentication/handler/authentication_handler.rb

```diff
@@ -31,8 +31,12 @@ def initialize(
       end
 
       def call(parameters:, request_ip:)
-        raise Errors::Authentication::RequestBody::MissingRequestParam, parameters[:code] unless parameters[:code]
-        raise Errors::Authentication::RequestBody::MissingRequestParam, parameters[:state] unless parameters[:state]
+        %i[state code].each do |parameter|
```

Comment:
Non-blocker: This could be made a little more self-documenting like:
```suggestion
        required_parameters = %i[state code]
        required_parameters.each do |parameter|
```